packages/react-ui/src/components/HatchifyDisplays/HatchifyDisplays.tsx

```diff
@@ -28,6 +28,17 @@ export const HatchifyExtraDisplay: React.FC<HatchifyExtraDisplayProps> = () => {
   return null
 }
 
+export type HatchifyEmptyListProps = {
+  label: string
+  after?: string
+} & ({ render: () => JSX.Element } | { ValueComponent: ValueComponent })
+
+export const HatchifyEmptyList: React.FC<HatchifyEmptyListProps> = () => {
+  return <div>No rows of data exist</div>
```

Comment:
i don't think this will ever render because on line #338 of `hatchifyDisplays` it looks like it's overwriting this default case, so I think this can be just be return null